src/coreComponents/mesh/generators/WellGeneratorBase.cpp

```diff
@@ -527,55 +528,71 @@ void WellGeneratorBase::debugWellGeometry() const
   {
     return;
   }
+  string sectionOutput;
+  string sectionName = "Section : Well generation";
+
+  sectionOutput = GEOS_FMT( "\n{:=^{}}\n", "=", sectionName.size() + 5 );
+  sectionOutput += GEOS_FMT( "{:^{}}\n", sectionName, sectionName.size() + 5 );
+  sectionOutput += GEOS_FMT( "{:=^{}}\n", "=", sectionName.size() + 5 );
+
+  std::cout << sectionOutput;
```

Comment:
We will need to have more log sections, this code need to be generalized later in a dedicated PR.